pepper-apis/src/main/java/org/broadinstitute/ddp/util/SendGridMailUtil.java

```diff
@@ -62,7 +68,7 @@ private static Mail buildEmailMessage(String fromName, String fromEmailAddress,
         return mail;
     }
 
-    private static void submitMailMessage(Mail mailMessage, String sendGridApiKey) {
+    private static void submitMailMessage(Mail mailMessage, String sendGridApiKey, String proxy) {
```

Comment:
A better: why don't we just use `SendGridClient.sendMail()` and handle the `ApiResult`? It pretty much does the same thing.